index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Survey form</title>
+    <link rel="stylesheet" href="styles.css">
+    
+</head>
+<body>
+    <div class="container">
+        <header class="heading">
+            <h1 id="title">freeCodeCamp Survey Form</h1>
+            <p class="survey-text" id="description">Thank you for the time to help us improve the platform</p>
+        </header>
+        <form action="" id="survey-form">
+            <div class="forminfo">
+                <label for="name" id="name-label">Name</label>
+                <input class="forms" type="text" id="name" placeholder="Enter your name" required>
+                
+            </div>
+            <div class="formifo">
+                <label for="email" id="email-label">E-mail</label>
```

Comment:
- [ ] Please note that you are expected to use the [GitHub inline comments feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) at least once to leave review comments.
* Please ensure that your comment is under a line or block of code that fits the context the comment is about.